src/browser/runtime_p2p/peer_connection_manager.ts

```diff
@@ -1,7 +1,21 @@
 import { EmitterMap } from './emitter_map';
 import { createHash } from 'crypto';
-import { connect, Identity, Fin } from 'hadouken-js-adapter';
+import { Identity, Fin } from 'hadouken-js-adapter';
 import { EventEmitter } from 'events';
+import * as path from 'path';
+
+/*
+    Try loading local dev js-adapter, if failed, resort to
```

Comment:
We do the [same thing in the renderer](https://github.com/HadoukenIO/core/blob/develop/src/renderer/main.js#L29)